apollo-router/src/executable.rs

```diff
@@ -26,7 +26,7 @@ pub struct Opt {
     /// Log level (off|error|warn|info|debug|trace).
     #[clap(
         long = "log",
-        default_value = "apollo_router=info,router=info,apollo_router_core=info,apollo_spaceport=info,tower_http=info,reqwest_tracing=info",
+        default_value = "apollo_router=info,router=info,apollo_router_core=info,apollo_spaceport=info,tower_http=info",
```

Comment:
Do we need to add `hyper=info` ?

Reply:
done in 2d94917a. I see the same spans generated